src/components/widgets/accounts-profit-and-loss/accounts-profit-and-loss.directive.coffee

```diff
@@ -46,6 +48,14 @@ module.controller('WidgetAccountsProfitAndLossCtrl', ($scope, $q, ChartFormatter
       $scope.period_translation = _.capitalize(translation.toLowerCase())
   )
 
+  # Display filter tags based on settings
+  # --------------------------------------
+  $scope.areFilterTagsEnabled = ->
```

Comment:
just a detail - In this case you can move it to `.init()` and assign a variable `$scope.areFilterTagsEnabled` => if you inject a method in the template, it's going to be evaluated at each digest cycle